posthog/models/event.py

```diff
@@ -187,31 +175,6 @@ def filter_by_element(self, filters: Dict, team_id: int):
 
         return {"elements_hash__in": groups.values_list("hash", flat=True)}
 
-    def filter_by_url(self, action_step: ActionStep, subquery: QuerySet):
-        if not action_step.url:
-            return subquery
-        if action_step.url_matching == ActionStep.EXACT:
-            where, param = "properties->>'$current_url' = %s", action_step.url
-        elif action_step.url_matching == ActionStep.REGEX:
-            where, param = "properties->>'$current_url' ~ %s", action_step.url
-        else:
-            where, param = "properties->>'$current_url' LIKE %s", f"%{action_step.url}%"
-        return subquery.extra(where=[where], params=[param])
-
-    def filter_by_event(self, action_step):
-        if not action_step.event:
-            return {}
-        return {"event": action_step.event}
-
-    def filter_by_period(self, start, end):
```

Comment:
there ought to be some tests doing similar things for the CH version too, right? Could you point me to those, couldn't find them.

(Just want to make sure new stuff still has tests for these basic cases)

Reply:
E.g. trends does something like this:

```python
    def _get_date_filter(self) -> Tuple[str, Dict]:
        date_filter = ""
        date_params: Dict[str, Any] = {}
        interval_annotation = get_trunc_func_ch(self._filter.interval)
        _, _, round_interval = get_time_diff(
            self._filter.interval, self._filter.date_from, self._filter.date_to, team_id=self._team_id
        )
        _, parsed_date_to, date_params = parse_timestamps(filter=self._filter, team_id=self._team_id)
        parsed_date_from = date_from_clause(interval_annotation, round_interval)
```